<!DOCTYPE html>
<html>
<head>
  <title>Multivariate Analysis in Metabolomics</title>
  <meta charset="utf-8">
</head>

<body>
  <h1><a href="index.html"
    title="Multivariate Analysis in Metabolomics">
    Multivariate Analysis in Metabolomics</a></h1>

  <ol>
    <li><a href="1.html">PCA</a></li>
    <li><a href="2.html">PLS</a></li>
    <li><a href="3.html">OPLS</a></li>
  </ol>

  <h3>Abstract</h3>
  <p>
  Metabolomics aims to provide a global snapshot of all small-molecule metabolites in cells and biological fluids, free of observational biases inherent to more focused studies of metabolism. However, the staggeringly high information content of such global analyses introduces a challenge of its own; efficiently forming biologically relevant conclusions from any given metabolomics dataset indeed requires specialized forms of data analysis. One approach to finding meaning in metabolomics datasets involves multivariate analysis (MVA) methods such as principal component analysis (PCA) and partial least squares projection to latent structures (PLS), where spectral features contributing most to variation or separation are identified for further analysis. However, as with any mathematical treatment, these methods are not a panacea; this review discusses the use of multivariate analysis for metabolomics, as well as common pitfalls and misconceptions.
  </p>

  <h3>Introduction</h3>
  Metabolomics is defined [1] as “the quantitative measurement of the multiparametric metabolic response of living systems to pathophysiological stimuli or genetic modification.” Such a definition implies that metabolomic studies offer the finest-grained detail available in the nascent field of systems biology: a molecular-level convolution of all upstream genomic, transcriptomic and proteomic responses of an organism to a given stimulus or change [2-4]. Metabolites are the end product of all cellular processes, and are a direct outcome of enzymatic and protein activity. Thus, metabolites are more proximal to a phenotype or disease than either genetic or proteomic information [5, 6]. This occurs because a simple change in the expression level of a gene or protein does not necessarily correlate directly with a variation in the activity level of a protein, but an alteration in a metabolite only occurs through such a change [7]. Consequently, metabolomics has been used to identify disease biomarkers [8, 9], to aid in the drug discovery process [10, 11], and to study plants [12], bacteria [13, 14], nutrition [15], and the environment [16], among numerous other applications [17].

  <br><br>
  <strong>Reference</strong><br>
  <u><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4465187/" target="_blank">
    https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4465187/</u>

</body>
</html>
