<!DOCTYPE html>
<html>
<head>
  <title>Multivariate Analysis in Metabolomics</title>
  <meta charset="utf-8">
</head>

<body>
  <h1><a href="index.html"
    title="Multivariate Analysis in Metabolomics">
    Multivariate Analysis in Metabolomics</a></h1>

  <ol>
    <li><a href="1.html">PCA</a></li>
    <li><a href="2.html">PLS</a></li>
    <li><a href="3.html">OPLS</a></li>
  </ol>

  <h3>Abstract</h3>
  <p>
  Metabolomics aims to provide a global snapshot of all small-molecule metabolites in cells and biological fluids, free of observational biases inherent to more focused studies of metabolism. However, the staggeringly high information content of such global analyses introduces a challenge of its own; efficiently forming biologically relevant conclusions from any given metabolomics dataset indeed requires specialized forms of data analysis. One approach to finding meaning in metabolomics datasets involves multivariate analysis (MVA) methods such as principal component analysis (PCA) and partial least squares projection to latent structures (PLS), where spectral features contributing most to variation or separation are identified for further analysis. However, as with any mathematical treatment, these methods are not a panacea; this review discusses the use of multivariate analysis for metabolomics, as well as common pitfalls and misconceptions.
  </p>

  <h3>Introduction</h3>
  Metabolomics is defined [1] as “the quantitative measurement of the multiparametric metabolic response of living systems to pathophysiological stimuli or genetic modification.” Such a definition implies that metabolomic studies offer the finest-grained detail available in the nascent field of systems biology: a molecular-level convolution of all upstream genomic, transcriptomic and proteomic responses of an organism to a given stimulus or change [2-4]. Metabolites are the end product of all cellular processes, and are a direct outcome of enzymatic and protein activity. Thus, metabolites are more proximal to a phenotype or disease than either genetic or proteomic information [5, 6]. This occurs because a simple change in the expression level of a gene or protein does not necessarily correlate directly with a variation in the activity level of a protein, but an alteration in a metabolite only occurs through such a change [7]. Consequently, metabolomics has been used to identify disease biomarkers [8, 9], to aid in the drug discovery process [10, 11], and to study plants [12], bacteria [13, 14], nutrition [15], and the environment [16], among numerous other applications [17].
  <br><br>
  However, metabolomics experiments are plagued with difficulty. The number of small-molecule metabolites in a biofluid, cell lysate, tissues or organ differs wildly depending on the organism studied, ranging from several hundred to hundreds of thousands [18]. Metabolomics is also a relatively new discipline and as a result a complete catalog of the human metabolome and the metabolomes of other organisms is not available [19]. Therefore, it is common to encounter unknown metabolites, a complication in the analysis and interpretation of metabolic changes. Similarly, the lack of reference NMR or MS spectra for all known metabolites makes proper identification of metabolites challenging [20-22]. Further, the exhibited diversity of chemical and physical properties of compounds within the metabolome makes true metabolomics (simultaneous quantification of all metabolites) unattainable with current instrumental capabilities [1, 18, 23]. As an illustration, the limited molecular-weight distribution of the metabolome prohibits a comprehensive and detailed analysis by mass spectroscopy and generally requires the additional use of chromatography [2, 24].
  <br><br>
  The analysis of metabolomic data is further complicated by the inherent variability in each sample. Every single cell, tissue, organ or organism is fundamentally unique [25], despite any defining feature they share in common, such as being the same species, infected with the same disease or receiving the same drug treatment. As such, the overall goal of metabolomics is to identify the few chemical features against a large and complex background of metabolites that uniquely define the system [20, 26]. These few chemical features or metabolites should be directly related to the defining characteristic of the system. But, unfortunately, all biological systems are easily perturbed by any number of experimental or environmental factors, such as age, diet, growth phase, media, nutrients, pH, sex, and temperature [27, 28]. Similarly, cell lysis, enzyme quenching and metabolome extraction techniques, and the storage of the metabolomics samples can also induce undesirable variations. There are also unavoidable fluctuations in spectral data, such as changes in peak position or peak width that are caused by instrument instability and variability in sample conditions. As a result, the analysis of metabolomic data requires a robust methodology to expose underlying trends in these highly complex and variable data sets.
  <br><br>
  One class of methods, appropriately termed “metabolic fingerprinting,” aims to retain much of the promised unbiased, global nature of the metabolomics experiment by differentially analyzing spectral information acquired from normal and perturbed systems [29]. Again, a perturbed system may result from a disease state, a drug treatment, the presence of a pest or parasite, an environmental stimulus (pH, temperature, toxin, or nutrient change), a temporal, spatial, or species difference. Metabolic fingerprinting is also amenable to high throughput data collection and analysis, since global metabolite profiling can be accomplished with minimal samples and rapid spectral acquisitions [30]. The goal of metabolic fingerprinting experiments is to determine the relative differences between the metabolomes of two or more systems to infer a biological relationship. Thus, a hallmark of metabolic fingerprinting is the use of multivariate analysis methods to identify those biologically relevant spectral features for further targeted analyses [1, 23, 31], with two of the most popular methods being principal component analysis (PCA) [32, 33] and partial least squares projection to latent structures (PLS) [34, 35]. In essence, PCA and PLS aim to differentiate between
  classes in highly complex data sets, despite within class variability. These multivariate analysis methods will be briefly described, along with a discussion of their application to metabolomics, with an emphasis on common errors and misconceptions.
  
  <br><br>
  <strong>Reference</strong><br>
  <u><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4465187/" target="_blank">
    https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4465187/</u>

</body>
</html> 
